src/snovault/elasticsearch/simple_queue.py

```diff
@@ -0,0 +1,156 @@
+'''
+Simple uuid queue for indexing process
+- Can be overridden by another script, class, module as long as it exposes
+the proper functions used in the indexer classes.
+'''
+import time
+from os import getpid
+
+
+class SimpleUuidServer(object):
+    '''Simple uuid queue as a list'''
+    def __init__(self):
+        self._uuids = []
+        self._client_conns = {}
+        self.errors = []
+
+    def _get_uuid(self):
```

Comment:
I think this technical should be a property object (https://www.programiz.com/python-programming/property)

Reply:
Possibly, these functions are 'stubs' for the more complex queue types, like redis or aws sqs.   It will be good to review this after seeing how the other queue types plug in.